docs/src/main/asciidoc/logging.adoc

```diff
@@ -302,8 +375,7 @@ For some, it will make sense to use logging that is humanly readable (unstructur
 
 ===== Configuration
 
-The JSON logging extension can be configured in various ways.
-The following properties are supported:
+Configure the JSON logging extension using supported properties to customize its behavior.
 
 include::{generated-dir}/config/quarkus-logging-json.adoc[opts=optional, leveloffset=+1]
```

Comment:
Does the restructuring and styling in this PR include the included file content? 
Note that this shows up in the preview with some grammatical mistakes.

Reply:
Hello @michelle-purcell . Yes, I will do that in a separate PR. My goal is to merge this asap as we get some conversation about the heading and abstract.
Before that happens, I will start patching this included files from a new branch.